kili/queries/label/__init__.py

```diff
@@ -314,3 +318,58 @@ def count_labels(
         result = self.auth.client.execute(GQL_LABELS_COUNT, variables)
         count = format_result("data", result)
         return int(count)  # type:ignore
+
+    def export_labels(
```

Comment:
I put it in the QueriesLabel because there is already the `export_labels_as_df function`. Does not have any impact when using on python it but it changes where the function reference is in the documentation. The other possibility would be in the QueriesProject.

Reply:
it's fine to put it here, to remain consistent with V2.